files-community/Files
# Fix: Improved the speed  graph

## Summary

Fixed some issues reported by other contributors.

## Task Checklist

- [x] Fixed an issue where the graph gets elevated when the graph reaches the maximum Y point.
- [x] Fixed an issue where it will incorrectly report a successful operation instead of marking it as canceled when selected one or more folders only and initiate compression, then cancel it,
- [x] Fixed an issue where it will erroneously report a failure instead of marking it as canceled when started decompression, then cancel it, it will indicate success instead of marking it as canceled when selected one or more folders with one or more files and start compression, then cancel it,

## Known Issues

There still are some room of string resource improvements but not the scope of this PR.

## PR Checklist

- [x] **Closes**: #13624

## Steps To Validate Changes

1. Launch the app
2. Go to folder content view
3. Copy files a lot
4. Open the Status Center
5. You now recognize the graph no longer moves up when the highest point updated.
6. You now recognize the cancelation state for compression/decompression is shown properly.

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
